### ISSUE TITLE
Add playlists to app

### ISSUE THREAD

**Issue by advait404** [metabrainz/listenbrainz-android#450]
Ho, it would be great if we can access playlists like 'weekly exploration' 'weekly jams' and others present in the 'created for you' section in the listenbrainz website.

**Comment by 07jasjeet** [comment]
Hi, @advait404, we have integrations of playlists coming up. We already have some pages up for testing on Beta.

**Comment by Sukrit-Prakash** [comment]
Hi ,   @07jasjeet  i want to contribute to this project, but i cant find relevant issues to fix. Can you tell me where to begin with..

**Comment by 07jasjeet** [comment]
Automatic playback from Spotify and Youtube Music has been added to roadmap, `Created For You` screen is live on v2.8.0 which has playlists :)

**Comment by ThreeDeeJay** [comment]
@07jasjeet So no built-in player? If so, perhaps it could just launch the YouTube URL externally so we can set apps that can auto-play it as default 🤔

**Comment by 07jasjeet** [comment]
@ThreeDeeJay In-built player is already present.

**Comment by ThreeDeeJay** [comment]
@07jasjeet Yeah, but I meant for new exploration/recommended tracks, which are most likely not available locally.
There's such an untapped potential for a practical mobile app that plays new/recommended tracks using our scrobble history for accuracy.
The closest I've come to that is just Spotify (which has limited scrobbling history) and ListenBrainz/Last.fm on my browser in desktop mode, but it's really clunky, especially because they both lack background playback controls other than Play/Pause (I'm guessing because the YouTube pop-up player doesn't know the next track) so it's tedious having to unlock and pull up the browser to skip/love/hate tracks (btw I think there should be some sort of "judging" mode where new/recommended tracks get auto-loved if played fully and auto-hated if skipped so they wouldn't be recommended again 👀👌)

**Comment by 07jasjeet** [comment]
@ThreeDeeJay Yes, my above comment of automatic remote playback mentions this feature set only :)
